releng/build.Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+pipeline {
+  agent any
```

Comment:
sure this will work on any agent?

Reply:
Any agent that lsp4j CI currently has. If (when!) it is moved to JIRO more will be needed here.